src/Transform/ONNX/ConstProp.cpp

```diff
@@ -217,6 +217,31 @@ struct ElementWiseBinaryOpImpl<ONNXMaxOp, T> {
   static T eval(T lhs, T rhs) { return std::max<T>(lhs, rhs); }
 };
 
+template <>
+struct ElementWiseBinaryOpImpl<ONNXModOp, int64_t, EnableNotBool<int64_t>> {
+  static int64_t eval(int64_t lhs, int64_t rhs) {
+    // The original calculation for mod
+    int mod = lhs - floor(lhs / rhs) * rhs;
```

Comment:
Minimally, `mod` should be an `int64_t` as returned. Explicit type lengths are preferred.

Reply:
Thanks for the correction.... just seeing that by accident :)

Reply:
Please push your changes, I don't see them yet.

Reply:
@AlexandreEichenberger Thanks!! There has been some connectivity issues today... the push was stalling. Thanks for the observation. What about now?